src/pages/workspace/taxes/ValuePage.tsx

```diff
@@ -0,0 +1,96 @@
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/WorkspaceTaxValueForm';
+
+type ValuePageProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.TAXES_VALUE>;
+
+function ValuePage({
+    route: {
+        params: {policyID, taxID},
+    },
+    policy,
+}: ValuePageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const currentTaxRate = PolicyUtils.getTaxByID(policy, taxID);
+    const [value, setValue] = useState(currentTaxRate?.value?.replace('%', ''));
+
+    const goBack = useCallback(() => Navigation.goBack(ROUTES.WORKSPACE_TAXES_EDIT.getRoute(policyID ?? '', taxID)), [policyID, taxID]);
+
+    const submit = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_TAX_VALUE_FORM>) => {
+            updatePolicyTaxValue(policyID, taxID, Number(values.value));
+            goBack();
+        },
+        [goBack, policyID, taxID],
+    );
+
+    if (!currentTaxRate) {
+        return <NotFoundPage />;
+    }
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policyID}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    shouldEnableMaxHeight
+                    testID={ValuePage.displayName}
+                >
+                    <HeaderWithBackButton
+                        title={translate('workspace.taxes.value')}
+                        onBackButtonPress={goBack}
+                    />
+
+                    <FormProvider
+                        formID={ONYXKEYS.FORMS.WORKSPACE_TAX_VALUE_FORM}
+                        submitButtonText={translate('workspace.editor.save')}
+                        style={[styles.flexGrow1]}
+                        scrollContextEnabled
+                        validate={validateTaxValue}
+                        onSubmit={submit}
+                        enabledWhenOffline
+                        disablePressOnEnter={false}
+                        shouldHideFixErrorsAlert
+                        submitFlexEnabled={false}
+                        submitButtonStyles={[styles.mh5]}
+                    >
+                        <InputWrapper
+                            InputComponent={AmountForm}
+                            inputID={INPUT_IDS.VALUE}
+                            defaultValue={value}
+                            onInputChange={setValue}
+                            hideCurrencySymbol
+                            extraSymbol={<Text style={styles.iouAmountText}>%</Text>}
```

Comment:
```suggestion
                            extraSymbol={<Text style={styles.iouAmountText}>%</Text>}
                            autoFocus
```

Reply:
Adding it doesn't make the autofocus work. I have to investigate it.